docs/config/aws_template.md

```diff
@@ -0,0 +1,440 @@
+
+### Open your `aws-template` repo in VSCode
+
+Open the `aws-template` repository in VSCode. You can either use the command line for this (i.e.,
+`code path/to/aws-template`), or just open it using ^^File -> Open Folder^^.
+
+???+ warning "Windows users"
+
+    Make sure you have the WSL extension by Microsoft
+    ([ms-vscode-remote.remote-wsl](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl)).
+    installed. Then press ++ctrl+shift+p++ and enter the command **`WSL: Reopen folder in WSL`**
+
+### Start your tsdat-cdk Docker container
+
+From your VSCode window, start a terminal (^^Main Menu -> Terminal -> New^^, OR you can press ++ctrl+grave++).
+
+Then from the VSCode terminal, run:
+
+ ```shell
+ docker build --platform linux/amd64 . -t tsdat-cdk # (1)!
+ docker compose up -d
+ ```
+
+1. In our testing we found that just `docker compose up -d` works fine on our team's Windows, Linux, and intel MacOS
+systems, but the `--platform` argument was needed for M1/M2 MacBooks. Milage may vary.
+
+### Attach a new VSCode window to the tsdat-cdk container
+
+1. Type the key combination:  ++ctrl+shift+p++ to bring up the VSCode command palette.
+1. Then from the input box type: "Dev-Containers:  Attach to Running Container..." and select it
+1. Then choose the  tsdat-cdk  container.
+
+This will start up a new VSCode window that is running from inside your tsdat-cdk container.
+
+### Open the provided cdk.code-workspace file
+
+From the VSCode window that is attached to the tsdat-cdk container click ^^Main Menu -> File-> Open Workspace from
+File^^. In the file chooser dialog, select `/root/aws-template/.vscode/cdk.code-workspace`
+
+!!! tip
+
+    A box should pop up in the bottom right corner that asks if you want to install the recommended extensions.
+    Select "Install".
+
+Once the extensions are installed, your workspace is ready! In the Explorer, you will see two top-level folders and a
+directory structure like so:
+
+1. :material-folder: **`aws-template/`**
+    * :material-folder: *`.vscode/`*
+    * :material-folder: *`.build_utils/`*
+    * :material-folder: `...`
+    * :material-file: *`pipelines_config.yml`*
+
+2. :material-folder: **`.aws/`**
+    * :material-file: *`config`*
+    * :material-file: *`credentials`*
+
+## Deploying your AWS Stack
+
+### Configure AWS/GitHub settings
+
+The top part of the `aws-template/pipelines_config.yml` contains settings related to the AWS-GitHub integration, where
+data should be pulled from & placed, and which AWS account should be used. Open this file and fill out the configuration
+options, using your own values as needed. This section only needs to be filled out once.
+
+```yaml title="aws-template/pipelines_config.yml"
+
+github_org: tsdat  # (1)!
+pipelines_repo_name: pipeline-template
+aws_repo_name: aws-template
+
+account_id: "XXXXXXXXXXX"  # (2)!
+region: us-west-2
+input_bucket_name: tsdat-input
+output_bucket_name: tsdat-output
+create_buckets: True
```

Comment:
Can we add another + info box for create_buckets:

If you have existing buckets that you would like to use for your pipeline inputs and outputs, then set create_buckets: False.  If create_buckets is set to True and the buckets already exist, then the deployment will throw an error.

Reply:
Adding this